src/util.h

```diff
@@ -1,30 +1,88 @@
 #include <fstream>
+#include <string.h>
+#include "solver.h"
 
-void file_to_grids(std::string filename, std::vector<Grid> &grids){
+
+struct StartupOptions
+{
+    std::string input_file = "simple-500";
```

Comment:
Maybe want to change this to a list. Then in the other file, just run through the whole loop.
If -in, then run that specific file, if not get all the files.